extern/ProtoGeometry/ProtoGeometry.config

```diff
@@ -1,4 +1,4 @@
 <?xml version="1.0"?>
 <ProtoGeometryConfiguration xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
-  <GeometryFactoryFileName>libg_220/LibG.ProtoInterface.dll</GeometryFactoryFileName>
+  <GeometryFactoryFileName>libg_219/LibG.ProtoInterface.dll</GeometryFactoryFileName>
```

Comment:
It looks like this adds all of the LibG ASM 219 files back. That sounds like a mistake...